index.js

```diff
@@ -26,6 +26,8 @@ const ServerResponse = require('fwsp-server-response');
 let serverResponse = new ServerResponse();
 const UMFMessage = require('fwsp-umf-message');
 
+const RedisConnection = require('./redis');
```

Comment:
That makes sense. However, I'm wondering how the reconnection strategies would be specified - in cases where it might be different from what's implemented.